test/blackbox-tests/test-cases/watching/watching-eager-concurrent-promote-command.t

```diff
@@ -0,0 +1,26 @@
+Demonstrate running "dune promote" concurrently with an eager rpc server
```

Comment:
We are using a passive rpc server here. But the distinction for us is not so important.